src/main/java/business/security/HttpSecurityConfiguration.java

```diff
@@ -86,7 +86,7 @@ protected void configure(HttpSecurity http) throws Exception {
                 .antMatchers(HttpMethod.GET, "/api/register/users/activate/**").permitAll()
                 .antMatchers(HttpMethod.GET, "/api/status").permitAll()
                 .antMatchers(HttpMethod.GET, "/api/ping").permitAll()
-                .antMatchers("/api/**").authenticated()
+                .antMatchers("/api/**", "/error").authenticated()
```

Comment:
Maybe better to set permitAll for '/error' endpoint? We configured it with permitAll in other projects